src/Module6/CustomConfigurationTest/Integration/ConfigurationManagerTests.cs

```diff
@@ -0,0 +1,22 @@
+using CustomConfiguration;
+using CustomConfigurationTest.Modules;
+using Xunit;
+
+namespace CustomConfigurationTest.Integration
+{
+    public class ConfigurationManagerTests
+    {
+        [Fact]
+        public void LoadConfiguration()
```

Comment:
The main idea of unit tests - it should not be dependent on anything, in your case TestConfiguration depends on "C:\Projects\SMT\src\SMT.Api\appsettings.json" , therefore tests are failing on my machine.

Reply:
Consider making unit tests independent!

Reply:
It's integration test